panos_upgrade_assurance/check_firewall.py

```diff
@@ -1372,3 +1399,89 @@ def check_device_root_certificate_issue(self, fail_when_affected_version_only: b
             "expire December 31st, 2023."
         )
         return result
+
+    def check_cdss_and_panorama_certificate_issue(self):
+        """Checks whether the device is affected by the following advisory;
+
+        https://live.paloaltonetworks.com/t5/customer-advisories/additional-pan-os-certificate-expirations-and-new-comprehensive/ta-p/572158
```

Comment:
Update docstring for pan.dev:
```suggestion
        """Checks whether the device is affected by the [PAN-OS Certificate Expirations Jan 2024 advisory][live-572158];

        [live-572158]: https://live.paloaltonetworks.com/t5/customer-advisories/additional-pan-os-certificate-expirations-and-new-comprehensive/ta-p/572158
```